Signal/src/ViewControllers/ConversationView/Cells/TypingIndicatorCell.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+import Foundation
+
+@objc(OWSTypingIndicatorCell)
+public class TypingIndicatorCell: ConversationViewCell {
```

Comment:
A simple cell that looks consistent with the message bubble cells, but is vastly simpler.  

The "real" message bubble cells are already way too complicated.  I didn't want to modify them to handle yet another special case.

Reply:
:+1: